examples/framework/preferences/minimalFootprint/js/loadScripts.js

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one of these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+var fluid_2_0_0 = fluid_2_0_0 || {};
+var fluid = fluid || fluid_2_0_0;
+
+(function (fluid) {
+    "use strict";
+
+    fluid.load = {};
```

Comment:
Should read fluid.load = fluid.load || {} since there is the barest chance that some day such material will end up in core and then loaded in the wrong order